ghdocs/BUILDING.md

```diff
@@ -21,37 +21,27 @@ The built files will be in the `/dist/` folder.
 
 ##Gulp tasks
 
-You can test and work with Fabric locally in a number of ways. 
+You can test and work with Fabric locally in a number of ways using the tasks listed below. 
 
 ###gulp
 
-Use this task to build Fabric once after you've made changes. This task will build all parts of Fabric and move all changed files into `/dist/`. Every time you make changes, re-run this task.
+Builds everything incrementally. Running this the first time will be slow but running it again will be really fast. This task will build all parts of Fabric and move all changed files into `/dist/`. Every time you make changes, re-run this task.
 
 ###gulp watch
 
-Use this tak to build Fabric after you make any changes. This task will build all parts of Fabric, and then rerun every time you make a change to any of the following folders: 
-- `/src/components/`
-- `/src/less/`
-- `/src/samples/`
-- `/src/templates/`
+Builds everything and watches everything. After running this once, your builds should be a lot faster. Now if you make a change anything anywhere in fabric, only that area/section/file will get build/changed/moved etc:
 
-###gulp watch:components-samples
+###gulp watch-debug
 
-Use this task to tweak and test components. This task will build only component samples and watch for changes within `/src/components/`. After the build runs, you can test your components by opening the `index.html` file located in any component inside `/dist/samples/Components/`.
+Builds everything and watches everything. After running this once, your builds should be a lot faster. This task is similar to gulp watch except you can now get a readout of what files are in the pipe, this will be helpful if you want to make sure the file you are working on get's picked up and added to the pipe.
```

Comment:
I'd do "Builds and watches" and "...files are in the pipe. This will be helpful..."

I think you've got a spare apostrophe in your get in "on get's picked up"